src/tests/queue-test/queue-test.c

```diff
@@ -64,8 +64,9 @@ void TimerFunction(osal_id_t timer_id)
 void task_1(void)
 {
     int32  status;
-    uint32 data_received;
     size_t data_size;
+    char   data_received[4] = {0};
+    char   expected[4]      = "xyz";
```

Comment:
I'd prefer if the data type stayed as `uint32` here.  It is more reflective of the way queues are actually used (most apps don't typically pass strings through queues - they are integer IDs or pointers more likely).

Reply:
do we want to revisit this?

Reply:
Data type was updated to be uint32, resolved.